config.py

```
@@ -1470,6 +1470,30 @@ def get_t1_from_meeg(bids_path):
     ```
 """
 
+source_info_update: Optional[Dict[str, str]] = dict(suffix='ave')
+"""
+Match the file name from where to read the info in forward and inverse solution
+computation by updating 'bids_path'.
+
+???+ example "Example"
+    ```python
+    source_info_update = {'processing': 'clean',
+                          'suffix': 'epo'}
+    ```
+"""
+
+inverse_targets: Optional[List[str]] = ['evoked']
+"""
+List of the different types of data on which to apply the inverse operator.
+
+If None only the inverse solution computation is done.
+
+???+ example "Example"
+    ```python
+    inverse_targets = ['raw', 'evoked', 'cov', 'epochs']
```

so far anything different from evoked with be silently ignored.